src/powershell/install-category.ps1

```diff
@@ -0,0 +1,12 @@
+$category = $env:CATEGORY
+Write-Output "Installing packages for category $category"
+
+[xml]$xml = Get-Content "D:\\packages\\$category.config"
+
+$xml.SelectNodes('//packages/package') | ForEach-Object {
```

Comment:
Is there a reason you used single quotes here or will this work?
```suggestion
$xml.SelectNodes("//packages/package") | ForEach-Object {
```

Reply:
@dav3r resolved with [85cbaed](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/85cbaed820d6399f84419b360ec22677e0369050)